### DISCUSSION TITLE
Yomitan style data field exports

### DISCUSSION THREAD

**Discussion by elronimous** [arianneorpilla/jidoujisho#399] [Ideas]
Update: Having the data export in the same formatting as Yomitan would allow for a universal Card Format / Styling to work between using Jidoujishou and Yomitan so i've updated the Discussion name to reflect this.
----
I would be wonderful to improve the formatting of the cards produced by Jidoujishou into a much more user-friendly format. In my opinion, Lorenzi's Jisho export with Html/CSS has nailed converting a JmDict entry into a nice format for flashcards.

https://jisho.hlorenzi.com/help/anki < You can view a sample here of the basics of what the anki formatting does. 

Do note that when exporting a List of words from Lorenzi's Jisho, it adds Html/CSS to the TSV that does things such as converting [n] to 'Noun' as well as highlighting titles appropriately etc. so that would need to happen during the 'send to ankidroid' stage of creating an entry.

**Comment by brishtibheja** [comment]
The conversation of n to nouns sounds great.

**Comment by elronimous** [comment]
I got nearly all of the way to this goal with CSS/HTML modificaton, but the export from the latest Jidoujisho [2.9.0-preview2] & JmDict appears to not export the word type headers (If I've missed something on this please let me know :) )

**Comment by elronimous** [comment]
@arianneorpilla 

Sorry to @ you but I was wondering if I was missing something for the Definitions from Jidoujisho. 
It seems that all formatting that comes out of the dictionary is broken for me, and ends up as pure text wit <br> .

Is there an option somewhere so it would export the same definition with the same parameters as say Yomitan would? This also applies for when it includes as a formatted table (Jidoujisho just turns them into <br> text lines

Eg Jidoujishou is:
![image](https://github.com/user-attachments/assets/787e3d65-1bcc-41c4-8c7d-5460f6cad94f)

Yomitan:
![image](https://github.com/user-attachments/assets/00f6c6bd-c0e4-4a03-8e75-3e793c111da1)

**Reply by ElaineGilstrom** [reply]
You aren't missing anything. I took at look at the code that creates the meaning field, and it compiles the field as plain text. It looks like it would be fairly easy to modify the code to reach parity with Yomitan in this regard, so once I get properly setup to contribute to the project I'll look into doing that. Need to learn dart and touch base with the existing team first though.
